pipelines/aggregation_merge.py

```diff
@@ -103,7 +103,7 @@ def merge(filepath):
 
                             binary_mask = (merged_tile != -9999).astype('int32')
                             eroded = ndimage.binary_erosion(binary_mask)
-                            boundary_tile &= binary_mask.astype(bool)
+                            boundary_tile &= eroded.astype(bool)
```

Comment:
This bug here was probably responsible for https://github.com/mapterhorn/mapterhorn/issues/93. Will check...